### PR TITLE
Added possibility to update user filter

### PR DESCRIPTION
**Pull request by mnicula**

### CODE REVIEW

File: src/ReportPortal.Client/Abstractions/Requests/UpdateUserFilterRequest.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        public string Description { get; set; }
+
+        /// <summary>
+        /// List of conditions to filter data.
+        /// </summary>
+        [DataMember(Name = "conditions")]
+        public IEnumerable<Condition> Conditions { get; set; }
+
+        /// <summary>
+        /// list of parameters of selection
```
Comment (nvborisenko) [inline src/ReportPortal.Client/Abstractions/Requests/UpdateUserFilterRequest.cs:34]: Should start from Upper case.

File: test/ReportPortal.Client.IntegrationTests/UserFilter/UserFilter.cs [hunk 2]
```diff
@@ -12,7 +12,7 @@ namespace ReportPortal.Client.IntegrationTests.UserFilter
     public class UserFilterFixture : BaseFixture
     {
         [Fact]
-        public async Task CreateGetDeleteUserFilter()
+        public async Task CreateGetUpdateDeleteUserFilter()
```
Comment (nvborisenko) [inline test/ReportPortal.Client.IntegrationTests/UserFilter/UserFilter.cs:15]: May I ask to create separate test to verify `Update` request only?

**Review by nvborisenko — CHANGES_REQUESTED** [review]
Thanks for the PR. Please fix inline comments. Correct token to make tests pass is `5c356bea-c2fb-4988-9a82-e63424859381`.

**Comment by mnicula** [comment]
Done

**Review by nvborisenko — APPROVED** [review]

**Pull request merged**